webdriver_wharf/db.py

```diff
@@ -10,6 +10,8 @@
 from sqlalchemy.orm import sessionmaker
 from sqlalchemy.ext.declarative import declarative_base
 
+from app import vnc_enabled, httpd_enabled
```

Comment:
This presents a small problem. app import db, and now db imports app. I'd  like to keep the import graph straightforward, so if possible I'd rather not see db import from app, and instead alter `interactions` to allow the corresponding values in `port_bindings` to be None, signaling that they shouldn't be forwarded.

The current port_bindings behavior was too tightly coupled with `interactions` even before this change, and it's possible that the correct solution is for this property to go away entirely, and have the port-forwarding decisions all be made in `interactions`.